timesketch/frontend-ng/src/views/Story.vue

```diff
@@ -30,12 +30,29 @@ limitations under the License.
       </v-card>
     </v-dialog>
 
+      <v-dialog v-model="deleteStoryDialog" max-width="300">
+        <v-card>
+          <v-card-title class="headline">Delete Story</v-card-title>
+          <v-card-text>
+            Are you sure you want to delete this story?
+          </v-card-text>
+          <v-card-actions>
+            <v-spacer></v-spacer>
+            <v-btn text @click="deleteStoryDialog = false">Cancel</v-btn>
+            <v-btn color="error" text @click="deleteStory">Delete</v-btn>
+          </v-card-actions>
+        </v-card>
+      </v-dialog>
+
     <v-hover v-slot="{ hover }">
       <v-toolbar dense flat class="mt-n3" color="transparent">
-        <v-toolbar-title @dblclick="renameStoryDialog = true"> {{ title }}</v-toolbar-title>
+        <v-toolbar-title> {{ title }}</v-toolbar-title>
         <v-btn v-if="hover" icon small @click="renameStoryDialog = true">
           <v-icon small>mdi-pencil</v-icon>
         </v-btn>
+        <v-btn v-if="hover" icon small @click="deleteStoryDialog = true" color="error">
+          <v-icon small>mdi-trash-can-outline</v-icon>
```

Comment:
Adding a title that shows on hover so it is clear what the button does.

```suggestion
          <v-icon small title="Delete Story">mdi-trash-can-outline</v-icon>
```